server/controllers/userInputController.ts

```diff
@@ -116,6 +121,10 @@ const createUserInputController = (
       const userId = res.locals.userId;
 
       logger.debug('Fetching input history for user:', userId);
+      // Check for authorization header. If not provided, throw 401 error
+      if (!req.headers.authorization) {
```

Comment:
The check for auth header in this controller is redundant here as it is already checked in the auth middleware.